__tests__/app.test.js

```diff
@@ -149,6 +149,41 @@ describe("GET /api/articles/:article_id/comments", () => {
 });
 })
 
+describe("GET /api/users", () => {
+  test("200: responds with an array of user objects, each with properties of username, name, and avatar_url ", () => {
+    return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then(({ body: { users } }) => {
+      expect(users).toHaveLength(4)
+      users.forEach((user) => {
+        expect(user).toMatchObject({
+          username: expect.any(String), 
+          name: expect.any(String),
+          avatar_url: expect.any(String)
+        })
+      })
+    })
+  })
+  // test("200: responds with an empty array when there are no users", () => {
+  //   return request(app)
+  //   .get("/api/users")
+  //   .expect(200)
+  //   .then(({ body: { users } }) => {
+  //     expect(users).toHaveLength(0)
+  //   })
+  // })
```

Comment:
No need to check for an error here - the only reason you would get no users if the database is down! And we don't need to test for that 👍 

Just the happy path test is enough :)